addons/mail/static/src/models/user_notification_manager.js

```diff
@@ -0,0 +1,137 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+
+import { url } from '@web/core/utils/urls';
+
+registerModel({
+    name: 'UserNotificationManager',
+    identifyingFields: [],
+    recordMethods: {
+        /**
+         * Send a notification, preferably a native one. If native
+         * notifications are disable or unavailable on the current
+         * platform, fallback on the notification service.
+         *
+         * @param {Object} param0
+         * @param {string} [param0.message] The body of the
+         * notification.
+         * @param {string} [param0.title] The title of the notification.
+         * @param {string} [param0.type] The type to be passed to the no
+         * service when native notifications can't be sent.
+         * @returns
+         */
+        sendNotification({ message, title, type }) {
+            if (!this.messaging.env.services['multi_tab'].isOnMainTab()) {
+                return;
+            }
```

Comment:
This looks like a functional change, if the notification was not native it was sent on the current tab no matter what (only the sound was prevented).